docs/elasticsearch.txt

```diff
@@ -94,6 +94,13 @@ After processing `ext`, JanusGraph checks for the following common options.  Jan
 * `index.[X].elasticsearch.ignore-cluster-name`
 * `index.[X].elasticsearch.health-request-timeout`
 
+===== Ingest Pipelines
+If using Elasticsearch 5.0 or higher, a different ingest pipelines can be set for each store.
+
+* `index.[X].elasticsearch.ingest-pipeline.[storeName]`
```

Comment:
What is would be the expected value of `storeName`? Is it the `my_pipeline_id` in the [ES example](https://www.elastic.co/guide/en/elasticsearch/reference/current/ingest.html) or is it the mixed index name?

Reply:
It's mixed index name, I change the doc to make it clearer.